Library/Formula/mesos.rb

```diff
@@ -12,19 +12,22 @@ class Mesos < Formula
   depends_on :java => "1.7+"
   depends_on :macos => :mountain_lion
   depends_on "maven" => :build
-  # Use our Zookeeper for Yosemite and not the one shipped with Mesos
-  # Remove with next release.
-  # See https://github.com/Homebrew/homebrew/issues/32965
-  depends_on "zookeeper" if MacOS.version == :yosemite
+  depends_on :apr => :build
+  depends_on "subversion"
+  
+
+  needs :cxx11
 
   def install
     args = ["--prefix=#{prefix}",
             "--disable-debug",
             "--disable-dependency-tracking",
             "--disable-silent-rules",
+            "--with-svn=/usr/local/Cellar/subversion/1.8.11",
```

Comment:
Make this non-prefix-subjective. ` #{Formula["subversion"].opt_prefix} `. (Means the formula won't break horribly if people have Homebrew installed in a non-/usr/local location). Apr is more fun. Don't set Apr unless we have to, It's complicated because of the way Apple packages Xcode and the CLT packages.